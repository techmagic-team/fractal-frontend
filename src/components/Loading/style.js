import styled from 'styled-components';

export const SpinnerWrapper = styled.div`
  position: absolute;
  width: 100%;
  height: 100%;
  top: 0;
  left: 0;
  display: flex;
  align-items: center;
  justify-content: center;
  background-color: rgba(255, 255, 255, 0.8);
`;
