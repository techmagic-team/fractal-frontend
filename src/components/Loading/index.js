import React from 'react';

import { SpinnerWrapper } from './style';

const Loading = () => (
  <SpinnerWrapper>
    <div className="spinner-border" role="status">
      <span className="sr-only">Loading...</span>
    </div>
  </SpinnerWrapper>
);

export default Loading;
