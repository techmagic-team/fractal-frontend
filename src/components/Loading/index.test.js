import React from 'react';
import { create } from 'react-test-renderer';

import Loading from './index';

describe('Loading', () => {
  it('should match the snapshot', () => {
    const component = create(<Loading />);

    expect(component.toJSON()).toMatchSnapshot();
  });
});
