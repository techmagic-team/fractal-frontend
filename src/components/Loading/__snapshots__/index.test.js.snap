// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading should match the snapshot 1`] = `
<div
  className="sc-bdVaJa jopxTi"
>
  <div
    className="spinner-border"
    role="status"
  >
    <span
      className="sr-only"
    >
      Loading...
    </span>
  </div>
</div>
`;
